src/TypeScriptGenerator.ts

```diff
@@ -48,11 +48,13 @@ const MODULE_IMPORT_FIELD = "MODULE_IMPORT_FIELD";
 const DIRECTIVE_NAME = "raw_response_type";
 
 export const generate: TypeGenerator["generate"] = (schema, node, options) => {
-  const ast: ts.Statement[] = IRVisitor.visit(
+  let ast: ts.Statement[] = IRVisitor.visit(
```

Comment:
can we avoid `let`?

Reply:
Why?

Reply:
Always eer on `const`. `let` is bug is intent-shift-prone.